src/cljs/zetawar/views.cljs

```diff
@@ -384,7 +384,12 @@
                                  :on-hide hide-settings}
      [:> js/ReactBootstrap.Modal.Header {:close-button true}
       [:> js/ReactBootstrap.Modal.Title
-       (translate :configure-faction-title)]]
+       (str "Configure Faction: "
+            (when @faction
```

Comment:
Nope, not missing anything, you're totally right. I'd go with the first option so that it's useful in contexts other faction configuration.